roles/ealapps/tasks/main.yml

```diff
@@ -7,6 +7,7 @@
     update: true
```

Comment:
Could we please also generate the hrc/admin/config.php file with the database credentials?

Here is the syntax of that file: https://github.com/PrincetonUniversityLibrary/video_reserves/blob/c72cd038b8f5d6155aebee75468a75bdbd834604/hrc/admin/config.php.tpl

Link to example tasks: https://github.com/pulibrary/princeton_ansible/blob/0abed277291c880aa9c6f4fb2349ac9e6e632586/roles/libwww/tasks/main.yml#L5-L19

Reply:
@kayiwa I just ran the playbook on the staging server and the `admin/config.php` file has not been created.

Reply:
Also, this comment should be for the video_reserves project - is it intentional that this PR has changes to both video_reserves and ealapps?